app/services/stats/r901_offender_sar_cases_report.rb

```diff
@@ -0,0 +1,62 @@
+require 'csv'
+
+module Stats
+  class R901OffenderSarCasesReport < BaseReport
+
+    CSV_COLUMN_HEADINGS = [
+        'Case number',
+        'Date received at MOJ',
+        'Final deadlinne', 
+        'Who is making the request?',
+        'Data subject name',
+        'Subject type',
+        'Page count',
+        'Timeliness (in time/out of time)',
+        'Case state',
+        'Times Taken',
```

Comment:
Not a big thing, but I think this should be 'Time taken' rather than 'Times taken' (unless it is a literal list of times i.e. 1pm, 2pm, 6pm, 7am)?

Time taken is a duration Times taken is a bit ambiguous to me.